src/contribute/docker-images.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+title: 'Beam docker images'
+permalink: /contribute/docker-images/
+---
+
+# Docker Images
+
+You can bootstrap or reproduce an Apache Beam developer environment by using
+our provided [docker scripts](https://github.com/apache/beam/tree/master/sdks/java/build-tools/src/main/resources/docker).
+In this directory you will find scripts to build and run docker images for
+different purposes:
+
+- **file**: Create a docker container from a .zip file with the Beam source
+  code in a given environment. It is useful to test/validate the release votes
+  or a specific version in a different version of Java than the host. It
+  reproduces a minimal docker image so it also isolate issues due to ports
+  shared in the machine, etc.
+
+- **git**: Same as file but the Beam source code comes from the git repository,
+  you can choose a given branch/tag/pull-request. Useful to test in a specific
+  environment.
+
+- **release**: It builds an end-user distribution of the latest version of Beam
+  and its dependencies. Useful for end-users who want to have a ready to use
+  container with Beam (Python only for the moment).
+
+## File based image
+
+If you want to build a container with a ready JDK 8 environment to test Beam:
+
+    cd file/openjdk8
```

Comment:
Yes indenting makes text as code, it is a convention of markdown, but I am going to add the block just to make it more explicit.